Admin Tools - Audit + Permissioning Documentation (#328)
hellohaptik/docs_haptik: docs/admin-tools/permission-management.md

---
title: Permission Management
---

The Permission management tool allows Partner Admins to manage the partners they have access to and provides the following functionality:

* Create New Partner
* Add Existing Users to a partner
* Create New User inside a partner
* Change Roles for the user
* Make Bulk Addition/Updates to the users

> All of the above functionality is only available to a Partner Admin.

Reviewer: ```suggestion
> All of the above functionality is only available to a Partner Admin.
```

## What is a Partner Admin?
A Partner Admin is a special role that can be assigned to someone who needs to manage the access and roles for all users under a partner.

## Available Roles in the System

> The roles are divided or categorised basis the tools they give access to. The available roles in the system are as follows:

Reviewer: ```suggestion
The roles are divided or categorised on the basis of tools. The available roles in the system are as follows:
```
